src/ImageProcessor.Web/Configuration/ImageProcessorConfiguration.cs

```diff
@@ -302,40 +302,18 @@ private void LoadImageServices()
                 }
 
                 IImageService imageService = Activator.CreateInstance(type) as IImageService;
-                if (!string.IsNullOrWhiteSpace(config.Prefix))
+                if (imageService != null)
                 {
-                    if (imageService != null)
-                    {
-                        imageService.Prefix = config.Prefix;
-                    }
+                    var name = config.Name ?? imageService.GetType().Name;
```

Comment:
Haha! If you've solved this in such a simple manner I think I may owe you a beer. I'll dl and have a good look asap. 👍